cli/src/pcluster/utils.py

```diff
@@ -580,18 +591,22 @@ def get_supported_os_for_scheduler(scheduler):
     :param scheduler: the scheduler for which we want to know the supported os
     :return: an array of strings of the supported OSes
     """
-    oses = ["alinux", "alinux2"]
-    if scheduler != "awsbatch":
-        oses.extend(["centos7", "centos8", "ubuntu1604", "ubuntu1804"])
-    return list(oses)
+    if scheduler == "awsbatch":
+        return ["alinux2"]
+    return SUPPORTED_OSES
```

Comment:
In future we should have a map in constants and retrieve the os according to the scheduler and os according to the architecture, rather than having a part in the constatns and a part here in the utility.

Reply:
Done